cli/test/lib/tasks/cache_spec.js

```diff
@@ -204,5 +204,21 @@ describe('lib/tasks/cache', () => {
       await cache.list()
       await snapshotWithHtml('second-binary-never-used.html')
     })
+
+    it('shows sizes', async function () {
+      sinon.stub(state, 'getPathToExecutable').returns('/.cache/Cypress/1.2.3/app/cypress')
+
+      const statAsync = sinon.stub(fs, 'statAsync')
+
+      statAsync.onFirstCall().resolves({
+        atime: moment().subtract(3, 'month').valueOf(),
```

Comment:
could you add an actual stats object to these mock folders, maybe even a recursive file to actually get folder size and not just 0.0?

Reply:
Done.